ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos.hs

```diff
@@ -423,11 +428,13 @@ instance PraosCrypto c => ConsensusProtocol (Praos c) where
   -- Updating the chain dependent state for Praos.
   --
   -- If we are not in a new epoch, then nothing happens. If we are in a new
-  -- epoch, we do two things:
+  -- epoch, we do three things:
   -- - Update the epoch nonce to the combination of the candidate nonce and the
   --   nonce derived from the last block of the previous epoch.
-  -- - Update the "last block of previous epoch" nonce to the nonce derived from
-  --   the last applied block.
+  -- - Store the current epoch nonce as the "previous epoch" nonce. This is
+  --   needed by Peras to be able to validate slightly old certificates.
```

Comment:
"slightly old" is not very precise, is there an issue or part of the design document we could refer to ?